Source/WebCore/loader/cache/CachedScript.cpp

```diff
@@ -150,17 +150,6 @@ void CachedScript::setBodyDataFrom(const CachedResource& resource)
 
 bool CachedScript::shouldIgnoreHTTPStatusCodeErrors() const
```

Comment:
Should we delete this whole function?

Reply:
I think no where, the only thing is that `shouldIgnoreHTTPStatusCodeErrors` in headers (.h) is marked as `final`. Hence, I didn't deleted it, others are `override` or `virtual` functions. If I remove this, it does compile still.

Reply:
@brentfulgham - Any thoughts or suggestions?

Reply:
Could It be called sometimes from  https://searchfox.org/wubkat/source/Source/WebCore/loader/SubresourceLoader.cpp

Reply:
I think we can safely remove because the definition is here - [CachedResource.h](https://searchfox.org/wubkat/source/Source/WebCore/loader/cache/CachedResource.h) and if we see hierarchy in searchfox.org, we can see it.

I will update this PR and remove this completely.